bin/cyhy-domainsync

```diff
@@ -43,10 +43,18 @@ def get_ipv4_addresses(hostname):
         answers = resolver.query(hostname, 'A')
         ip_list = [IPAddress(answer.address) for answer in answers]
         return set(ip_list)
-    except exception.DNSException as e:
-        # Could not resolve domain name
-        return set()
+    # Could not resolve domain name
     except AddrFormatError:
+        print("    Warning: Address format error for " + hostname)
+        return set()
+    except resolver.NoAnswer:
+        print("    Warning: DNS NoAnswer (no A records found) for " + hostname)
+        return set()
+    except resolver.NXDOMAIN:
+        print("    Warning: DNS NXDOMAIN (no domain exists) for " + hostname)
```

Comment:
Do we want to include the output from the `responses` method as defined in https://www.dnspython.org/docs/1.16.0/dns.resolver.NXDOMAIN-class.html?